Cargo.toml

```diff
@@ -62,9 +62,9 @@ ruint = { version = "1.12.3", default-features = false, features = ["alloc"] }
 # External dependencies #
 #########################
 anyhow = "1.0"
-alloy = { version = "0.12.5", default-features = false }
-foundry-common-fmt = { git = "https://github.com/foundry-rs/foundry", rev = "v1.0.0", package = "foundry-common-fmt" }
-foundry-macros = { git = "https://github.com/foundry-rs/foundry", rev = "v1.0.0", package = "foundry-macros" }
+alloy = { version = "1.0.7", default-features = false }
+foundry-common-fmt = { git = "https://github.com/foundry-rs/foundry", rev = "ab753e9cafc5937bcc868fd7c61237c34ef9ac74", package = "foundry-common-fmt" }
+foundry-macros = { git = "https://github.com/foundry-rs/foundry", rev = "ab753e9cafc5937bcc868fd7c61237c34ef9ac74", package = "foundry-macros" }
```

Comment:
I am like 99.99% sure we don't actually need foundry in dependencies anymore but I'll get rid of it in a follow-up PR. This has been on my todo list for quite a while.

Reply:
This was a big PITA during the upstream merge, yup. These things are [somehow used in foundry-zksync](https://github.com/matter-labs/foundry-zksync/blob/main/crates/zksync/core/src/vm/decoder.rs#L6), but I guess they can simply be moved there (and probably it can be even simplified).